File: Examples/test-suite/php/cbooltest_runme.php
<?php

require "tests.php";

check::equal(cbooltest::do_or(true, false), true, "do_or() test failed");
check::equal(cbooltest::do_and(true, false), false, "do_and() test failed");
check::equal(cbooltest::TRUE_CONSTANT, true, "TRUE_CONSTANT test failed");
check::equal(cbooltest::FALSE_CONSTANT, false, "FALSE_CONSTANT test failed");

File: Examples/test-suite/python/cbooltest_runme.py
import cbooltest

if not cbooltest.do_or(True, False):
    raise RuntimeError("bad _Bool support")

if cbooltest.do_and(True, False):
    raise RuntimeError("bad _Bool support")

if not cbooltest.TRUE_CONSTANT:
    raise RuntimeError("bad _Bool support")

if cbooltest.FALSE_CONSTANT:
    raise RuntimeError("bad _Bool support")
